tests/unit/tags.t

```diff
@@ -627,34 +627,36 @@ is_deeply(
 # removing 2 tags in a list of 4
 $product_ref = {
 	lc => "fr",
-	categories => "pommes, bananes, en:pears, fr:fraises, es:limones",
+	categories => "pommes, bananes, pears, fraises",
 };
 
-remove_tag_from_field($product_ref, "fr", "categories", "bananes, fraises");
-is_deeply($product_ref->{categories}, "pommes, oranges");
+# my @tags_to_remove = split(/,\s*/, "bananes, fraises");
+
+# remove_tags_from_field($product_ref, "fr", "categories", @tags_to_remove);
+remove_tags_from_field($product_ref, "fr", "categories", "bananes, fraises");
```

Comment:
it's not the way you function work… please read about parameters in functions, and lists in perl. You are messing things.

Reply:
You mean this way `remove_tags_from_field($product_ref, "fr", "categories", "bananes, fraises");`?

Reply:
Sorry I think it's me who misread :blush: